compose-common.yml

```diff
@@ -29,3 +29,33 @@ base:
     # We enable Python hash randomization to protect Django from certain DoS attacks.
     # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/#python-options
     - PYTHONHASHSEED=random
+
```

Comment:
If above `# These settings probably shouldn't need a change` is valid for the whole rest of the file, it should be un-indented (if YAML allows that)